docs/fundamentals/clean-code.md

```diff
@@ -263,4 +263,64 @@ with f:
 ```
 ````
 
+## Power heavy-lifting job by the Flow, not Client
+   
+Wrap heavy lifting jobs inside an `Executor` and part of the `Flow`.
+For instance, loading high resolution images from Client into `Document` and `DocumentArray`
+could be time consuming.
+Put it inside an Executor could leverage Jina's capability to scale it up.
+
+````{tab} ✅ Do
+```{code-block} python
+---
+emphasize-lines: 12
+---
+import glob
+
+from jina import Executor, Flow, requests, Document, DocumentArray
+
+class MyExecutor(Executor):
+
+    @requests
+    def to_blob_conversion(self, docs: DocumentArray, **kwargs):
+        for doc in docs:
+            doc.convert_image_uri_to_blob()  # conversion happens inside Flow
+
+f = Flow().add(uses=MyExecutor)
+
+def _load_data():
+    da = DocumentArray()
+    image_uris = glob.glob('/.workspace/*.png')  # load high resolution images.
+    for image_uri in image_uris:
+        doc = Document(uri=image_uri)
+        da.append(doc)
+    return da
+
+with f:
+    f.post('/foo', inputs=_load_data())
+```
+````
+
+````{tab} 😔 Don't
+```{code-block} python
+---
+emphasize-lines: 12
+---
```

Comment:
note that this will highlight the line number 12, starting from line 278 (considered number 1)
So unless you have another important line to highlight, I suggest you replace with:
```suggestion
```python
```